backend/.env

```diff
@@ -1,3 +1,3 @@
 # PORT=8000
-# HOST=0.0.0.0
+HOST=0.0.0.0
```

Comment:
This should be commented and ENV should be added to `backend/Dockerfile`

```
WORKDIR /app/backend
ENV HOST=0.0.0.0
```

Reply:
Corrected